Makefile

```diff
@@ -22,7 +22,7 @@ dockerpullall:
 	docker pull phpmyadmin/phpmyadmin:latest
 	docker pull silintl/data-volume:latest
 	docker pull silintl/mariadb:latest
-	docker pull silintl/php7:latest
+	docker pull silintl/php7:7.2
```

Comment:
It turns out that there is also a `docker-compose pull` that will automatically pull the relevant images, so this whole make command could be simplified to just that, if desired.

Reply:
With that knowledge, I think I'll remove that target.